types/got/got-tests.ts

```diff
@@ -343,6 +343,11 @@ got('http://todomvc.com', { timeout: 1 }).catch((err) => err instanceof got.Time
 // Test hooks.
 got('example.com', {
     hooks: {
+        init: [
```

Comment:
As you can see each test contains only one case, i.e. hook. So, move this hook test to separate case.